virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,29 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 
 [id="virt-2-4-known-issues"]
 == Known issues
+
+* If container-native virtualization 2.3 is installed on your {product-title} 4.4 cluster, upgrading the cluster to version 4.5 causes a migrating virtual machine instance (VMI) to fail when the target node is evicted during the upgrade.
```

Comment:
"causes a migrating virtual machine instance (VMI) to fail" - should be "may cause"
"when the target node is evicted during the upgrade." should be removed.